docs/matlab_differences.rst

```diff
@@ -18,6 +18,31 @@ internal math.
     :depth: 3
 
 
+Differences in Signal Detrending
+--------------------------------
+
+In the PREP pipeline, trends (i.e., slow drifts in EEG baseline signal) are
+temporarily removed from the data prior to line-noise removal with CleanLine
```

Comment:
can we get a link to CleanLine here? or some further info in brackets about it?

Reply:
Sure, I'll add a link to the relevant MNE function.

Reply:
I decided to remove the reference to CleanLine and just put "adaptive line-noise removal". Since the relevant MNE function defaults to a different sort of filter, adding a link and explaining the correct args was just going to make that sentence unwieldly. Hopefully that works!